discord_slash/context.py

```diff
@@ -57,6 +59,7 @@ def __init__(self,
             self.author = discord.User(data=_json["member"]["user"], state=self.bot._connection)
         else:
             self.author = discord.User(data=_json["user"], state=self.bot._connection)
+        self.received_at = datetime.now()  # the time this context was created (used for cooldowns)
```

Comment:
I think for consistency with dpy, this should be renamed to `self.created_at`. Additionally, rather than relying on the system time which could be inaccurate, and may not be in UTC time, a better option would be to use `discord.utils.snowflake_time` as is [used across the discord.py lib](https://github.com/Rapptz/discord.py/blob/a8f44174bafed3989ec2959a62b89006f4a9e9a1/discord/role.py#L244).

Reply:
Agreed, committing